lib/Client.js

```diff
@@ -286,6 +287,7 @@ class Client extends EventEmitter {
     */
     addMessageReaction(channelID, messageID, reaction, userID) {
         if(userID !== undefined) {
+            emitDeprecation("eris:1", "addMessageReaction() was called without an \"@me\" `userID` argument");
             this.emit("warn", "[DEPRECATED] addMessageReaction() was called without an \"@me\" `userID` argument");
```

Comment:
Shouldnt these be removed?

Reply:
oop wrong line i meant the emit warn

Reply:
No, we're keeping them for now for backwards compatibility, but we can remove them in a future breaking change. See the summary I wrote in #1274.

Reply:
Ah I see thats fair, kinda ironic that the deprecations are deprecated tho lol